packages/react/src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -15,16 +27,21 @@ export interface LabelVariables {
 export default (siteVars: SiteVariablesPrepared): LabelVariables => {
   const color = 'rgba(0, 0, 0, 0.6)'
 
+  const colorScheme = extendColorScheme(siteVars.colorScheme, {
+    default: {
+      background: color,
+      foreground: 'rgb(232, 232, 232)',
+    },
+    brand: {
```

Comment:
not sure if you were going to address this by the set of changes provided, but as a client I would expect that `brand` would appear under props of `LabelColorSchemeMapping`, so that at least types would suggest me that this is a safe thing to use:

![image](https://user-images.githubusercontent.com/9024564/57776419-1ec2f080-7728-11e9-9a20-26a3916ef2dd.png)

Reply:
The problem is that now, we are using one ColorNames type, but this should be redefine for teams theme (we don't have all colors that the typings suggest, and we have few more. Will introduce this changes too.

Reply:
Introduced TeamsColorNames, a bit scared as how these typings with generics are looking now :S I am open for ides for restructuring them. Will try on my own to see how can I improve these.